_gsocproposals/2020/proposal_IntAlertSystem.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Intelligent Alert Management System For HEP experiments
+layout: gsoc_proposal
+project: Monitoring
+year: 2020
+organization: CERN
+---
+
+## Description
+
+We would like to build an intelligent Alert Management System for HEP experiments.
+It should embrace existing open-source solutions based on Prometheus and AlertManager
+to select alerts from various sub-systems and reduce amount
```

Comment:
"sub-systems" is quite vague: is it about online subsystems, distributed computing nodes, ...?